src/test/resources/features/complaints/ComplaintsTriage.feature

```diff
@@ -139,6 +150,18 @@ Feature: Complaints Triage
     And the summary should display the owning team as "Minor Misconduct"
     And an Escalation note should be visible in the timeline showing the submitted reason for the cases escalation
 
+  @ComplaintsWorkflow @ComplaintsRegression
+  Scenario: User is able to escalate a BF case to workflow manager at the Case Triage stage
+    Given I am logged into "CS" as user "BF_USER"
+    When I create a "BF" case and move it to the "Triage" stage
+    And I load and claim the current case
```

Comment:
Please use 'I get a "BF" case at the "Triage" stage'. Then you wont require the second step about loading and claiming the case.